update_actions.sh

```diff
@@ -0,0 +1,44 @@
+
+#!/bin/bash
+set -e
+
+VERSION=$1
+
+if [ -z "$VERSION" ]; then
+        echo "missing version"
+        exit 1
+fi
+
+actionsRepos=(okteto/delete-namespace
+              okteto/build
+              okteto/destroy-preview
+              okteto/deploy-preview
+              okteto/deploy-stack
+              okteto/namespace
+              okteto/pipeline
+              okteto/push
+              okteto/create-namespace
+              okteto/destroy-pipeline
+              okteto/login
+              okteto/destroy-stack
+              okteto/apply)
+              
+for repo in "${actionsRepos[@]}"
+do
+    echo "$repo"
+    git clone --depth 1 https://github.com/$repo.git
+    repoPath=${repo#"okteto/"}
+    echo $repoPath
+    pushd $repoPath
+    git config user.name "okteto"
+    git config user.email "ci@okteto.com"
+    sed -iE 's_FROM\ okteto\/okteto\:latest_FROM\ okteto\/okteto\:'$VERSION'_' Dockerfile
+    sed -iE 's_FROM\ okteto\/okteto\:[[:digit:]]*\.[[:digit:]]*\.[[:digit:]]*_FROM\ okteto\/okteto\:'$VERSION'_' Dockerfile
+    git add Dockerfile
+    git commit -m "$VERSION release"
+    git push git@github.com:$repo.git master
+    git --no-pager log -1
+    ghr -u ${CIRCLE_PROJECT_USERNAME} -token $GITHUB_TOKEN -replace $VERSION
```

Comment:
will this release also update the marketplace? (when you go through the UI, you need to mark a specific checkbox)

Reply:
No, I'm searching the way but GitHub API docs is not providing any way of doing this...

Reply:
What's the state for this? Ideally, we want to publish it to the marketplace too